### ISSUE TITLE
re-assigning the navigatorObject_moveFocus script to nvda+numpadplus for one-hand handling of the navigator

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2965]
**Reported by dave090679 on 2013-02-04 17:23**
hi guys,
I tink the navigator would be more consitent if it could be handled with only one hand.
To do so, the hnavigatorObject_moveFocus script should be assigned sto nvda+numpadplus instead of nvda+shift+numpadminus.

In the attached gestures.ini I re-assgned the nvda+numpadplus key combination to navigatorObject_moveFocus and I deleted the assignment of nvda+shift+numpadminus.

happy testing.

Dave

**Comment by ehollig** [comment]
Was this issue posted prior to the change in the keyboard layout? From the what's new, it looks like the changes in the keyboard commands took place in 2013.1. NVDA+Shift+NumPadPlus does not currently do anything. Personally, I think the commands are fine how they are, and if someone would like to change them for one reason or another, they can change it in the input gestures dialog. Thoughts? CC @feerrenrut @jcsteh

**Comment by jcsteh** [comment]
Although I accept that the current keystroke cannot be executed with one hand, I think the risk of breaking backwards compatibility for existing users outweighs the gain here. Note that this can now be easily reassigned using the Input Gestures dialog, which did not exist at the time this issue was filed. Closing as won't fix.
